commit 00522598eab59b612c7ba31b77deb4258c8af009
Author: Ben Kohler <bkohler@gentoo.org>
Date:   Mon Mar 29 09:06:33 2021 -0500

    x11-wm/spectrwm: remove incorrect EROOT usage
    
    Package-Manager: Portage-3.0.18, Repoman-3.0.3
    Signed-off-by: Ben Kohler <bkohler@gentoo.org>

diff --git a/x11-wm/spectrwm/spectrwm-3.4.1.ebuild b/x11-wm/spectrwm/spectrwm-3.4.1.ebuild
index 3b826527316..3b297f77149 100644
--- a/x11-wm/spectrwm/spectrwm-3.4.1.ebuild
+++ b/x11-wm/spectrwm/spectrwm-3.4.1.ebuild
@@ -36,12 +36,12 @@ src_prepare() {
 
 src_compile() {
 	tc-export CC PKG_CONFIG
-	emake -C linux PREFIX="${EROOT}/usr" LIBDIR="${EROOT}/usr/$(get_libdir)"
+	emake -C linux PREFIX="/usr" LIBDIR="/usr/$(get_libdir)"
 }
 
 src_install() {
-	emake -C linux PREFIX="${EROOT}/usr" LIBDIR="${EROOT}/usr/$(get_libdir)" \
-		SYSCONFDIR="${EROOT}/etc" DOCDIR="${EROOT}/usr/share/doc/${P}" \
+	emake -C linux PREFIX="/usr" LIBDIR="/usr/$(get_libdir)" \
+		SYSCONFDIR="/etc" DOCDIR="/usr/share/doc/${P}" \
 		DESTDIR="${D}" install
 
 	dodoc README.md ${PN}_*.conf {initscreen,screenshot}.sh
